File: addons/terrabrush/src/CustomContentPieMenu.cs
using Godot;

namespace TerraBrush;

[GodotClass(Tool = true)]
public partial class CustomContentPieMenu : Control {
    private PieMenu _pieMenu;

    public PieMenu PieMenu => _pieMenu;

    protected override void _Ready() {
        base._Ready();

        _pieMenu = new PieMenu() {
            ButtonsMinRadius = 20
        };
        AddChild(_pieMenu);
    }
}

File: src/editor_nodes/custom_content_pie_menu.cpp
#include "custom_content_pie_menu.h"

using namespace godot;

void CustomContentPieMenu::_bind_methods() {

}

CustomContentPieMenu::CustomContentPieMenu() {

}

CustomContentPieMenu::~CustomContentPieMenu() {}

void CustomContentPieMenu::_ready() {
    _pieMenu = memnew(PieMenu);
    _pieMenu->set_buttonsRadiusRatio(20);
    add_child(_pieMenu);
}

PieMenu *CustomContentPieMenu::get_pieMenu() const {
    return _pieMenu;
}